oat-sa/extension-tao-testqti
# feat/AUT-1612/allow set QTI Identifier pattern from configuration

Related to: https://oat-sa.atlassian.net/browse/AUT-1612
Depends on: 

- [x] https://github.com/oat-sa/extension-tao-itemqti/pull/2007
- [x] update in composer version of taoQtiItem

Added support to update QTI Identifier pattern from configuration for Item and Test Authoring

**How to test:**
1. set in config new pattern `config/tao/client_lib_config_registry.conf.php`
```
        'taoQtiItem/qtiCreator/widgets/helpers/qtiIdentifier' => array(
            'qtiIdPattern' => '^[a-zA-Z_][a-zA-Z0-9_-]*$'
        )
```
2. check that in Item and Test Authoring QTI identifier for test, test-part, test-section and assessmentItemRef with dot is not valid (error message will stay the same)
